recipes/python-keystoneclient/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "python-keystoneclient" %}
+{% set version = "3.15.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "f82c43c1cde7453ee274090c2e816c27c8a4c1de88583a8a5a509719917396fa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
Could this recipe possibly be `noarch: python`?

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages